### PR TITLE
completed html forms

### PR DESCRIPTION
**Pull request by hnguy96**
Screen Shot 2021-03-09 at 3 14 38 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Form</title>
+</head>
+<body>
+  <form action="">
+    <div>
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      Email
+    </div>
+    <label for="email">
+      <input type="text">
+    </label>
+    <div>
```
Comment (codingcodymiller) [inline html-forms/index.html:17]: This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <option value="because it's powerful">Because it's powerful</option>
+      </select>
+    </label>
+    <div>
+      Favorite Language (choose one)
+    </div>
+    <label for="favorite language">
+      <input type="radio" name="language" value="html" /> HTML
+      <input type="radio" name="language" value="css" /> CSS
+      <input type="radio" name="language" value="javascript" /> JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:34]: The text for each option should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    </label>
+    <div>
+      Would you like to sign up for more information?
+    </div>
+    <label for="sign up for more information">
+      <input type="checkbox" value="sign me up" /> Sign me up!
+    </label>
+    <div>
```
Comment (codingcodymiller) [inline html-forms/index.html:42]: This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      Comments
+    </div>
+    <label for="comments">
```
Comment (codingcodymiller) [inline html-forms/index.html:45]: This `label` element can be removed

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    <div>
+      Email
+    </div>
+    <label for="email">
+      <input type="text">
+    </label>
+    <div>
+      Why is coding cool?
+    </div>
+    <label for="why is coding cool">
```
Comment (codingcodymiller) [inline html-forms/index.html:20]: This `label` element can be removed

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Form</title>
+</head>
+<body>
+  <form action="">
+    <div>
+      Email
+    </div>
+    <label for="email">
```
Comment (codingcodymiller) [inline html-forms/index.html:14]: This `label` element can be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes

File: html-forms/index.html (hunk 1 above)
Comment (hnguy96) [reply html-forms/index.html:11]: So a `form` element should have a `label` element wrapped around it as well?

**Review by codingcodymiller — APPROVED** [review]
Nicely done!!!

**Pull request merged**